packages/mongodb-memory-server-core/src/util/MongoBinary.ts

```diff
@@ -32,54 +32,65 @@ export interface MongoBinaryOpts {
 export default class MongoBinary {
   static cache: MongoBinaryCache = {};
 
+  /**
+   * Probe if the provided "systemBinary" is an existing path
+   * @param systemBinary The Path to probe for an System-Binary
+   * @return System Binary path or empty string
+   */
   static async getSystemPath(systemBinary: string): Promise<string> {
     let binaryPath = '';
 
     try {
       await promisify(fs.access)(systemBinary);
 
-      log(`MongoBinary: found sytem binary path at ${systemBinary}`);
+      log(`MongoBinary: found sytem binary path at "${systemBinary}"`);
       binaryPath = systemBinary;
     } catch (err) {
-      log(`MongoBinary: can't find system binary at ${systemBinary}. ${err.message}`);
+      log(`MongoBinary: can't find system binary at "${systemBinary}".\n${err.message}`);
     }
 
     return binaryPath;
   }
 
-  static async getCachePath(version: string): Promise<string> {
+  /**
+   * Check if specified version already exists in the cache
+   * @param version The Version to check for
+   */
+  static getCachePath(version: string): string {
```

Comment:
changed this function to not being async anymore, because it never needed to be async